pyiron_base/project/generic.py

```diff
@@ -15,6 +15,7 @@
 from git import Repo, InvalidGitRepositoryError
 
 from pyiron_base.project.path import ProjectPath
+from pyiron_base.project.factory import Creator
```

Comment:
Rather than having `generic/factory.py`, I feel like it makes more sense to put the factory files next to the definitions of their objects. This will prevent having a million factories being crammed into one generic factory file in modules where there's lots of factories. It also makes them easier to find, but that is maybe just my opinion.

With this, I think `Creator` can simply be implemented right in `project.py` as it's quite short and directly related.

Reply:
Done